### PR TITLE
Set DATASET_URL default and enable to run out of the box

### PR DESCRIPTION
**Pull request by lresende**
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

**Comment by yil532** [comment]
@lresende @ptitzler Should dataset link be https://dax-cdn.cdn.appdomain.cloud/dax-noaa-weather-data-jfk-airport/1.1.4/noaa-weather-data-jfk-airport.tar.gz ?

File: pipelines/dax_noaa_weather_data/load_data.ipynb [hunk 1]
```diff
@@ -74,10 +74,7 @@
    },
    "outputs": [],
    "source": [
-    "if os.environ.get('DATASET_URL') is None:\n",
-    "    raise RuntimeError('Cannot run notebook. Required parameter is missing.')\n",
-    "    \n",
-    "data_file = os.environ['DATASET_URL']"
+    "data_file = os.getenv('DATASET_URL', 'https://dax-cdn.cdn.appdomain.cloud/dax-fashion-mnist/1.0.2/fashion-mnist.tar.gz')"
```
Comment (ptitzler) [inline pipelines/dax_noaa_weather_data/load_data.ipynb:77]:
```suggestion
    "data_file = os.getenv('DATASET_URL', 'https://dax-cdn.cdn.appdomain.cloud/dax-noaa-weather-data-jfk-airport/1.1.4/noaa-weather-data-jfk-airport.tar.gz ')"
```

**Review by ptitzler — CHANGES_REQUESTED** [review]
Would be curious to understand how hardwiring the data set URL helps, considering that we are positioning `load_data` as a generic download node for DAX data sets. It doesn't save the user any time or make things simpler
- the pipeline file already defines all required inputs
- if one wants to re-create the pipeline from scratch one still has to define the other dependencies

**Comment by lresende** [comment]
> Would be curious to understand how hardwiring the data set URL helps, considering that we are positioning `load_data` as a generic download node for DAX data sets. It doesn't save the user any time or make things simpler
> 

I just want to make the notebook runnable by itself, if the user is not running via a pipeline.

**Review by ptitzler — APPROVED** [review]
LGTM

**Pull request merged**